environment.devenv.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - eigen
   - catch2
   - gxx_linux-64=7.3.0  # [linux]
+  - clangxx_osx-64=9.0.0  # [osx]
```

Comment:
Allan, I think that maybe you can try `- compilers` instead. This is not a recommended approach, but there is a chance that this way works.

Reply:
However, I don't know how `autodiff`'s cmake is configured. If it's configured to use `clang`, so the above comment will not work.

Reply:
> However, I don't know how `autodiff`'s cmake is configured. If it's configured to use `clang`, so the above comment will not work.

Ok, laziness is not a good thing (unless in lazy evaluations). So I will break my laziness and have a look in cmake as soon as I have some time.

Reply:
I added this `clangxx_osx-64` to ensure that when compiling for macOS (e.g., in Travis), we use the same compiler that conda-forge is using. However, Travis is exporting `CXX` to its own compiler - I need to overwrite this.

How is this `- compilers` supposed to work?

Reply:
Ah, `- compilers` will get a full set of compilers compatible to your system _and_ conda. You can check it [here](https://anaconda.org/conda-forge/compilers), but unfortunately without details.

Reply:
Cool, so I can remove those two lines and use `compilers` instead, right?

Reply:
Yeah, at least I think so!